spec/factories/users.rb

```diff
@@ -2,9 +2,9 @@
   factory :user do
     name { Faker::Name.name }
     email { Faker::Internet.safe_email }
-    password 'test'
-    password_confirmation 'test'
-    created_at Time.now
-    updated_at Time.now
+    password { 'test' }
+    password_confirmation { 'test' }
+    created_at { Time.now }
+    updated_at { Time.now }
```

Comment:
__[RuboCop]__ <sub>posted by Sider</sub>

Rails/TimeZone: Do not use `Time.now` without zone. Use one of `Time.zone.now`, `Time.current`, `Time.now.in_time_zone`, `Time.now.utc`, `Time.now.getlocal`, `Time.now.xmlschema`, `Time.now.iso8601`, `Time.now.jisx0301`, `Time.now.rfc3339`, `Time.now.httpdate`, `Time.now.to_i`, `Time.now.to_f` instead.